Helper/Settings.php

```diff
@@ -211,6 +212,18 @@ public function getNewsletterList($storeId = null)
         return $this->getSettingsVal(self::XML_NEWSLETTER_LIST_VALUE, $storeId);
     }
 
+    /**
+     * Is remove customer in sailthru
+     *
+     * @param $storeId
+     *
+     * @return mixed
+     */
+    public function isRemoveUserInSailthru($storeId = null)
```

Comment:
- Rename `isRemoveUserInSailthru` to `shouldRemoveCustomerDataInSaulthru`
- Add `(bool)` type casting to the return value